picodom.d.ts

```diff
@@ -1,26 +1,41 @@
-export interface VNode<Data> {
+export as namespace picodom
+
+export interface VirtualNode<Data> {
   tag: string
-  data?: Data
+  data: Data
   children: VElement<{}>[]
 }
 
 export interface Component<Data> {
-  (data?: Data, ...children: VElement<{}>[]): VNode<Data>
+  (data: Data, children: VElement<{}>[]): VirtualNode<any>
 }
 
-export type VElement<Data> = VNode<Data> | string
+export type VElement<Data> = VirtualNode<Data> | string
+
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  children?: Array<VElement<{}> | number>
```

Comment:
`h` accepts an array of children. This enables line 7 in h-test.ts in this PR. I've also added number as an acceptable type to `h` to enable line 14 of h-test.ts.